exercises/pov/description.md

```diff
@@ -1,7 +1,5 @@
 Reparent a graph on a selected node.
 
-# Tree Reparenting
-
```

Comment:
I get the sense that at one point, perhaps, the exercise was intended to be called Tree Reparenting.  I don't think anything is lost by removing this heading.